.github/workflows/iroha2-dev.yml

```diff
@@ -46,9 +46,8 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
-# FIXME: This workflow never worked properly. Should we remove it? 
```

Comment:
Actually there is not problem in `load-rs` gob here. Since it just trigger `dispatch` workflow from other repository, that `oad-rs-ci:dev` workflow in [soramitsu/iroha2-longevity-load-rs](https://github.com/soramitsu/iroha2-longevity-load-rs/actions/runs/3765611744/jobs/6401283080) should be fixed.